wom/src/main/scala/wom/callable/ContainerizedInputExpression.scala

```diff
@@ -5,7 +5,7 @@ import wom.expression.IoFunctionSet
 import wom.values.WomValue
 
 /**
-  * This is an expression that
+  * This is an expression that is all I can say.
```

Comment:
Not strictly necessary but I'd be 👍 on fellow reviewers trying to fill in this blank with something else :)

Reply:
At the very least. please put a TODO here to fix this this comment before handover

Reply:
If the original author would like to suggest some text I'd be happy to include it in this PR. 😛

Reply:
Original author here!  Could you please use `that uses "containerized" (a.k.a. "mapped" input values) to evalute its expression`.